roles/manage-ssh-config-root-access/README.md

```diff
@@ -0,0 +1,48 @@
+manage_ssh_config_root_access role
+=========
+
+This role updates `/etc/sshd_config` to deny ssh password authentication for root.
```

Comment:
Overall, I'd just make this a `manage-sshd-config` and not tie it to just one specific task. I can see us adding additional features/functionality to this in the future as well.

Reply:
BTW: note the `d` at the end of `ssh`